packages/lesswrong/components/editor/EditorFormComponent.tsx

```diff
@@ -201,12 +202,17 @@ export const EditorFormComponent = ({form, formType, formProps, document, name,
     ${getFragment('RevisionEdit')}
   `);
 
+  // TODO: this currently clobbers the title if a new post had its contents edited before the title was edited
```

Comment:
Can you explain this to me? Currently what happens is that, if a user makes a new post, and then edits the contents, but then edits the title, the title won't be saved? Seems bad?

Reply:
Only happens with that order of operations via autosave (which this PR also disables for now, per https://lworg.slack.com/archives/CJUN2UAFN/p1705016179980999?thread_ts=1705000179.503459&cid=CJUN2UAFN)